go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
+	gopkg.in/yaml.v2 v2.2.8 // indirect
```

Comment:
any idea where this came from? says it's indirect but just curious which of these changes might have brought it in

Reply:
I do not know the answer to this question. I did compile this code using Go 1.13.6 and can confirm that the `go.mod` and `go.sum` files are clean. I also verified `go mod tidy` and it is good.